config/routes.rb

```diff
@@ -11,6 +11,7 @@
     patch 'users/:user_id/add_patient/:id', to: 'users#add_patient', as: 'add_patient', defaults: { format: :js }
     patch 'users/:user_id/remove_patient/:id', to: 'users#remove_patient', as: 'remove_patient', defaults: { format: :js }
     post 'users/search', to: 'users#search', as: 'users_search', defaults: { format: :js }
+    patch 'users/clear_current_user_call_list', to: 'users#clear_current_user_call_list', as: 'clear_current_user_call_list', defaults: { format: :js }
```

Comment:
i think it should be this? ```users/:user_id/clear_current_user_call_list```

Reply:
I think we're looking for it to either hook into an `:id` (in which case we should do the db lookup to get whatever user corresponds to that id), or assume that it relies on `current_user` (which is whoever is logged in, and I think is a safe assumption). Works either way, but should be one or the other.